apps/src/dance/ai/utils.ts

```diff
@@ -0,0 +1,29 @@
+import {BlockSvg, Workspace} from 'blockly';
+
+/**
+ * Generates blocks from the AI result in a given workspace,
+ * and attaches them to each other.
+ */
+export const generateBlocksFromResult = (
+  workspace: Workspace,
+  resultJsonString: string
```

Comment:
Probably a refactor for another time, but it would nice to pass the parsed result object here for the sake of type safety, so we're not having to stringify and parse so much as we pass this data between components?